src/Pos/Wallet/Web/Api.hs

```diff
@@ -108,103 +108,103 @@ type TestReset =
     :> WRes Post ()
 
 -------------------------------------------------------------------------
--- Wallet sets
+-- Wallets
 -------------------------------------------------------------------------
 
-type GetWalletSet =
+type GetWallet =
        "wallets"
     :> "sets"
     :> Capture "walletSetId" (CId Wal)
     :> WRes Get CWallet
 
-type GetWalletSets =
+type GetWallets =
        "wallets"
     :> "sets"
     :> WRes Get [CWallet]
 
-type NewWalletSet =
+type NewWallet =
        "wallets"
     :> "sets"
     :> "new"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] CWalletInit
     :> WRes Post CWallet
 
-type RestoreWalletSet =
+type RestoreWallet =
        "wallets"
     :> "sets"
     :> "restore"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] CWalletInit
     :> WRes Post CWallet
 
-type RenameWalletSet =
+type RenameWallet =
        "wallets"
     :> "sets"
     :> "rename"
-    :> Capture "walletSetId" (CId Wal)
+    :> Capture "walletId" (CId Wal)
     :> Capture "name" Text
     :> WRes Post CWallet
 
-type DeleteWalletSet =
+type DeleteWallet =
        "wallets"
     :> "sets"
     :> Capture "walletSetId" (CId Wal)
     :> WRes Delete ()
 
-type ImportWalletSet =
+type ImportWallet =
        "wallets"
     :> "sets"
     :> "keys"
     :> CQueryParam "passphrase" CPassPhrase
     :> ReqBody '[JSON] Text
     :> WRes Post CWallet
 
-type ChangeWalletSetPassphrase =
+type ChangeWalletPassphrase =
        "wallets"
     :> "sets"
     :> "password"
-    :> Capture "walletSetId" (CId Wal)
+    :> Capture "walletId" (CId Wal)
     :> CQueryParam "old" CPassPhrase
     :> CQueryParam "new" CPassPhrase
     :> WRes Post ()
 
 -------------------------------------------------------------------------
--- Wallets
+-- Accounts
 -------------------------------------------------------------------------
 
-type GetWallet =
+type GetAccount =
        "wallets"
     :> CCapture "walletId" CAccountId
     :> WRes Get CAccount
 
-type GetWallets =
+type GetAccounts =
        "wallets"
-    :> QueryParam "walletSetId" (CId Wal)
+    :> QueryParam "walletId" (CId Wal)
```

Comment:
It has to reflect the change in BackendApi.purs https://github.com/input-output-hk/cardano-sl/blob/9411eb79c76a68b551620fe4574c7a257516684d/daedalus/src/Daedalus/BackendApi.purs#L140